packages/litexa-deploy-aws/test/manifest.spec.coffee

```diff
@@ -20,11 +20,23 @@ chai.use(chaiAsPromised)
 
 
 describe 'construct and deploy artifacts for manifest', ->
+  this.timeout(5000)
```

Comment:
Maybe add a comment as to what this is for?

Reply:
done